salt/_modules/vault.py

```diff
@@ -372,6 +418,12 @@ def configure_auth_methods(self, client, remote_methods, local_methods):
                 )
             except hvac.exceptions.InvalidPath:
                 pass
+            except NotImplementedError:
+                log.debug(
+                    'No methods defined to retrieve extra configuration for auth method "%s"...',
+                    auth_method.type,
+                )
+                pass
```

Comment:
No need to `pass` in this case. If you do not re-raise in the `except` block, the exception is not raised.